drivers/serial/uart_npcx.c

```diff
@@ -367,6 +386,10 @@ static int uart_npcx_init(const struct device *dev)
 #endif
 
 #if defined(CONFIG_PM)
```

Comment:
Really would like to use IS_ENABLED() if possible?

Reply:
In this section, the driver registers a callback function, [uart_npcx_rx_wk_isr](https://github.com/zephyrproject-rtos/zephyr/pull/32776/files#diff-80314ccee38f06d27252b7ac2d69a582d3687c8b71808737fdbbb0bc31182fd9R298), in [line 391](https://github.com/zephyrproject-rtos/zephyr/pull/32776/files#diff-80314ccee38f06d27252b7ac2d69a582d3687c8b71808737fdbbb0bc31182fd9R391) for handling the related wake-up input event. If this PR replaces "#ifdef" with "IS_ENABLED()" here, we will get a linking error, "'uart_npcx_rx_wk_isr' undeclared", if CONFIG_PM is not enabled. That's why I didn't choose IS_ENABLED() here.

Reply:
Done.